api/src/components/awaken/awaken.routes.js

```diff
@@ -0,0 +1,8 @@
+const express = require('express');
+const awakenController = require('./awaken.controller');
+
+const awakenRouter = express.Router();
+
+awakenRouter.get('/awaken', awakenController.get);
```

Comment:
I'm tempted to not use the 'controller' way of doing things as it can get very MVC sounding. See above comment for more :)